build.sbt

```diff
@@ -10,11 +10,12 @@ import de.heikoseeberger.sbtheader.HeaderPattern
 
 val PlayVersion = "2.5.0"
 val AkkaVersion = "2.4.4"
-val AkkaPersistenceCassandraVersion = "0.13"
+val AkkaPersistenceCassandraVersion = "0.17"
```

Comment:
This PR https://github.com/lagom/lagom/pull/134 (which was merged already) takes care of updating the akka-persistence-cassandra version